content/en/dora_metrics/setup/deployments.md

```diff
@@ -0,0 +1,210 @@
+      }
+    }
+  }
+EOF
+```
+
+[1]: /api/latest/dora-metrics/#send-a-deployment-event-for-dora-metrics
+[2]: https://app.datadoghq.com/organization-settings/api-keys
+{{% /tab %}}
+
+{{% tab "datadog-ci CLI" %}}
+
+The [`datadog-ci`][1] CLI tool provides a shortcut to send deployment events within your Continuous Integration environment.
+
+For the following example, set the `DD_SITE` environment variable to {{< region-param key="dd_site" code="true" >}} and set the `DD_API_KEY` environment variable to your [Datadog API Key][2]:
+```shell
+export DD_BETA_COMMANDS_ENABLED=1
+export DD_SITE="<DD_SITE>"
+export DD_API_KEY="<API_KEY>"
+
+export deploy_start=`date +%s`
+./your-deploy-script.sh
+datadog-ci dora deployment --service shopist --env prod \
+    --started-at $deploy_start --finished-at `date +%s` \
+    --git-repository-url "https://github.com/organization/example-repository" \
+    --git-commit-sha 66adc9350f2cc9b250b69abddab733dd55e1a588
+```
+
+The deployment finish time is automatically set to now if `--finished-at` is not provided.
+
+If the deployment CI job is running on the exact same Git revision that is being deployed, `git-repository-url` and `git-commit-sha` can be omitted and will be automatically inferred from the CI context.
+The `--skip-git` option can be provided to disable sending the repository URL and commit SHA. When this option is added, the Change Lead Time metric will not be available.
+
+[1]: https://www.npmjs.com/package/@datadog/datadog-ci
+[2]: https://app.datadoghq.com/organization-settings/api-keys
+{{% /tab %}}
+{{< /tabs >}}
+
+## Calculating Change Lead Time
+
+For a single Git commit, the Change Lead Time (CLT) is calculated as the time from the commit creation to when the deployment including the commit change was executed.
+To calculate the Change Lead Time for a deployment, Datadog automatically finds all the commits done since the previous deployment, and computes the average of the related Change Lead Time values.
+Change Lead Time is not available for the first deployment of a service that includes Git information.
+
+For the Change Lead Time metric to be available, two requirements need to be met:
+1. The Git repository URL and the commit SHA information are provided when sending deployment events.
+2. Your repository metadata is being [synchronized to Datadog](#synchronize-repository-metadata-to-datadog).
+
+### Synchronize repository metadata to Datadog
+
+<!--
+The Following tabs were mostly copied from the Source Code Integration docs until we find a way to document this in a shared page
+https://docs.datadoghq.com/integrations/guide/source-code-integration/?tab=github#synchronize-your-repository-metadata
+-->
+
+<div class="alert alert-info">
+The retention of Git metadata is 1 month. Commits older than 1 month might not be taken into account when computing Change Lead Time. Datadog doesn't store the actual content of files in your repository, only Git commit and tree objects.
```

Comment:
can we put the text below into a "Limitations" sub-section at the bottom of the page? 

"The retention of Git metadata is 1 month. Commits older than 1 month might not be taken into account when computing Change Lead Time. Change Lead Time is not available for the first deployment of a service that includes Git information."

and remove the blue callout (blue + orange banners at once is a bit too much IMO). "Datadog doesn't store the actual content of files in your repository, only Git commit and tree objects" can go at the end of the initial paragraph under "Calculating change lead time"

Reply:
Changed! Let me know if it works